consensus/fork/galactica.go

```diff
@@ -18,11 +18,13 @@ import (
 )
 
 var (
+	// errBaseFeeTooHighForLegacyTx is returned if the base fee is too high for a legacy tx
+	errBaseFeeTooHighForLegacyTx = errors.New("base fee too high for legacy tx, use dynamic fee tx or retry later")
```

Comment:
do you think we can have a similar error naming? Since they refer to the same root cause. Like `errBaseFeeTooHighForLegacyTx` and `errBaseFeeTooHighForDynamicFeeTx`

Reply:
Done, thanks